internal/cmdrender/executor.go

```diff
@@ -59,52 +66,71 @@ func (e *Executor) Execute(ctx context.Context) error {
 
 	// initialize hydration context
 	hctx := &hydrationContext{
-		root:            root,
-		pkgs:            map[types.UniquePath]*pkgNode{},
-		fnResults:       fnresult.NewResultList(),
-		imagePullPolicy: e.ImagePullPolicy,
+		root:             root,
+		pkgs:             map[types.UniquePath]*pkgNode{},
+		fnResults:        fnresult.NewResultList(),
+		imagePullPolicy:  e.ImagePullPolicy,
+		disableCLIOutput: disableCLIOutput,
 	}
 
 	resources, err := hydrate(ctx, root, hctx)
 	if err != nil {
 		// Note(droot): ignore the error in function result saving
 		// to avoid masking the hydration error.
-		_ = e.saveFnResults(ctx, hctx.fnResults, true)
+		_ = e.saveFnResults(ctx, hctx.fnResults, true, disableCLIOutput)
```

Comment:
in case of error, we do want to print out where the results are saved. And as discussed, add the test case to cover that.

Reply:
made it false.